src/dbos-runtime/config.ts

```diff
@@ -181,8 +181,15 @@ export function parseConfigFile(cliOptions?: DBOSCLIStartOptions, useProxy: bool
   /*************************************/
   /* Build final runtime Configuration */
   /*************************************/
+  const entrypoints = new Set<string>();
+  // CLI overrides configuration
```

Comment:
Didn't we decide to remove this from the CLI?

Reply:
good catch, forgotten comment here.